ash-window/src/lib.rs

```diff
@@ -118,6 +118,50 @@ pub unsafe fn create_surface(
     }
 }
 
+/// Extensions necessary for creating a surface on the target platform.
```

Comment:
```suggestion
/// Extensions necessary for creating a surface on the current target platform.
```

Again, I feel like making it very clear that the value of this `const` is going to change depending on the compile target, and it doesn't feel like just mentioning "target platform" gives that enough weight.